model/histogram/float_histogram_test.go

```diff
@@ -966,8 +981,8 @@ func TestFloatHistogramCompact(t *testing.T) {
 			&FloatHistogram{
 				PositiveSpans:   []Span{{-2, 10}},
 				PositiveBuckets: []float64{1, 3.3, 0, 0, 0, 0, 0, 4.2, 0.1, 3.3},
-				NegativeSpans:   []Span{{0, 9}},
-				NegativeBuckets: []float64{3.1, 3, 0, 0, 0, 1.234e5, 1000, 3, 4},
+				NegativeSpans:   []Span{{0, 7}},
+				NegativeBuckets: []float64{3.1, 3, 0, 1.234e5, 1000, 3, 4},
```

Comment:
That's not correct. In the original histogram, the second span has an offset to the previous span of 3. That's reduced here to only 1. Or in other word: The test was correct before.